website/docs/zkvm/quickstart.md

```diff
@@ -42,14 +42,27 @@ Don't worry -- each `TODO` comes with instructions for what you need to do, and
 
 ## 4. Run your project, locally or remotely
 
-The readme for your project includes instructions for how to run it, both locally or remotely.
-You can build your example and run the prover locally with:
+The readme for your project includes instructions for how to run it, both locally or remotely. You can also get a better developer experience and smoother iteration on your code by using the developer mode.
+
+### Running your project
+
+To run your project locally, going through the full proof generation process, run:
 
 ```bash
 cargo run --release
 ```
 
-> **Congratulations!** <br/>_That's all it takes to build and run a minimal RISC Zero application._
+#### Tip: Leveraging Developer Mode
```

Comment:
```suggestion
#### Tip for Quick Development: Leveraging Developer Mode 
```
I think it would make this easier to read if we tell the readers what they can accomplish using this mode

Reply:
Completely agree, thanks for the input!